### DISCUSSION TITLE
Bticino Living Now via Gateway Controller HomeKit

### DISCUSSION THREAD

**Discussion by alessio-savino** [indomus/forum#1732] [HUB: Home Assistant]
Ciao a tutti, giovedì scorso alle 15.54 tutti i miei dispositivi livingnow (luci e tapparelle) sono diventati improvvisamente "non disponibili", sono collegate tramite Gateway Controller HomeKit e hanno sempre funzionato... dai log non vedo nulla, semplicemente sono non disponibili. Sapete se ci log specifici da verificare o dove posso indagare?

**Comment by Salsalove** [comment]
A me è successo oggi...tutti non disponibili. Come hai risolto (se hai risolto)? @alessio-savino

**Reply by alessio-savino** [reply]
Tolto la corrente al gateway e riacceso. Sono ricomparsi magicamente

**Reply by Salsalove** [reply]
ok, grazie...ci provo!
